src/main/build/debug.c

```diff
@@ -107,4 +107,6 @@ const char * const debugModeNames[DEBUG_COUNT] = {
     "ATTITUDE",
     "VTX_MSP",
     "GPS_DOP",
+    "ANGLEMODE",
+    "ANGLE_FEEDFORWARD",
```

Comment:
```suggestion
    "ANGLE_DERIVATIVE",
```

Reply:
I think it's better to have P and D on one debug and FF on another so we can easily tune PD balance.